src/service.ts

```diff
@@ -46,10 +44,6 @@ export class KeycloakService {
     }
 
     this.connect = keycloak as Keycloak
-    this.client = new AdminClient({
```

Comment:
Can you provide a solution without removing the reference to `AdminClient` since this would break a lot of applications.